README.md

```diff
@@ -1,2 +1,454 @@
+```
+
+# 4. Reserved Capabilities
+
+The following capabilities are REQUIRED to be implemented.
+
+## 4.1 All Owned Resources
+
+### 4.1.1. `my` Scheme
+
+The `my` URI scheme represents ownership over a resource -- typically by parenthood -- at decision-time (i.e. the validator's "now"). Resources that are created after the UCAN was created MUST be included. This higher-order scheme describes delegating some or all ambient authority to another DID.
+
+The use case of "pairing" two DIDs by delegating all current and future resources is not uncommon when a user would like to use multiple devices as "root", but does not have access to all of them directly at all times.
+
+The format for this scheme is as follows:
+
+``` abnf
+ownershipscheme = "my:" kind ["@" did]
+kind = "*" / <scheme> 
+```
+
+The wildcard `*` resource MUST be taken to mean "everything" (all resources of all types).
+
+A "sub-scheme" MAY be used to delegate some of that scheme controlled by parenthood. For example `my:dns` delegates access to all DNS records (`my:dns:*` works equally well since wildcards are also part of the DNS URI). `my:mailto` selects all owned email addresses controlled by this user.
+
+Re-delegating these to further DIDs in a chain MUST address the specific parent DID that owns that resource separated by an `@`. For instance: `my:*@did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` selects all resources originating from the specified DID, and `my:mailto@did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` selects email addresses from the DID. 
+
+### 4.1.2 Action
+
+The action for `my:*` MUST be `*`.
+
+``` json
+{"with": "my:*", "can": "*"}
+```
+
+For `my` capabilities scoped to some scheme, the action MUST be one normally associated with that resource. 
+
+``` json
+{"with": "my:dns", "can": "crud/UPDATE"}
+```
+
+## 4.2 UCAN Addressing
+
+### 4.2.1 `prf` Scheme
+
+The `prf` URI scheme defines addressing for UCANs and their fields.
+
+``` abnf
+prf = "prf" selector
+selector = "*" / 1*DIGIT
+```
+
+`prf:*` represents all of the UCANs in the current proof scope. The witnesses in the `prf` field may be referenced by their index, starting at 0: `prf:0`
+
+### 4.2.2 `prf` Actions
+
+The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This re-delegates all of the capabilities in the selected witness(es).
```

Comment:
> Personally I thought empty att meant exactly that as in 0 constraints/restrictions had been added.

It's the other way around! An empty `att` means that the issuer is delegating _nothing:_ the `aud` gains nothing but to know that you _have_ these capabilities at your disposal. (We use this as part of the device pairing AKE). You need to explicitly "re-export" all of the capabilities that you want to give to the `aud`.

>  maybe it would make more sense to have separate "delegate" field to list indices into "prf" or encode delegation as number in the "att" or different shaped capability would be a better option.

We looked at this previously (not sure where the issue went 🤔) and landed on encoding these using the normal capability system.


>  {
>   "att": [
>      { "with": "prf:0:mailto:*", "can": "msg/*" },
>      { "with": "mailto:boris@fission.codes", "can": "msg/SEND" }
>   ]
> }

I don't think that `"prf:0:mailto:*"` is going to be a super common use case. If you need that, you can explicitly delegate the relevant capability. That should be made clearer in the spec 👍